src/redis/Redis.cpp

```diff
@@ -594,10 +572,8 @@ void Redis::setLong(const std::string& key, long value)
 {
     // Format is NOT printf compatible contrary to what the docs say, hence %i
     // instead of %l.
-    auto reply =
-      (redisReply*)redisCommand(context, "SET %s %i", key.c_str(), value);
-
-    freeReplyObject(reply);
+    auto reply = wrapReply(
+      (redisReply*)redisCommand(context, "SET %s %i", key.c_str(), value));
```

Comment:
Does this not cause an unused variable warning? Can we now just call `wrapReply` and not do any cleanup?

Reply:
Yes, I was batch-fixing this and didn't notice the specific cases here, will fix in a sec